crates/activity_indicator/src/activity_indicator.rs

```diff
@@ -28,6 +28,7 @@ pub enum Event {
 pub struct ActivityIndicator {
     statuses: Vec<LspStatus>,
     project: Model<Project>,
+    ignored_direnv_worktrees: HashSet<WorktreeId>,
```

Comment:
I think it'd be nicer if this whole file didn't have any reference of Direnv. "environment errors" or something like that, seems better.

Reply:
Sure, makes sense